configator/main.py

```diff
@@ -0,0 +1,44 @@
+"""This file will use all of our functions and run the program"""
+
+import typer
+
+# Import from configator
+from configator import create_actions_config
+from configator import create_gatorgradle_yml
+from configator import generate_build_gradle
+
+#pylint: disable=too-many-arguments
+app = typer.Typer()
+@app.command()
+def main(
+
+    name: str = typer.Option("configator-generated"),
+    brk: str = typer.Option("true"),
+    fastfail: str = typer.Option("false"),
+    ind: int = typer.Option(2),
+    vers: str = typer.Option("master"),
+    ggradleversion: str = typer.Option("0.5.1"),
```

Comment:
You should use better names for these options, like `version` or `indent` instead of these abbreviated names. In the case that you want a name with a dash in it, you can instead pass a second parameter to `typer.Option` like: `typer.Option("true", "--break")` or `typer.Option("master", "--grader-version")`.